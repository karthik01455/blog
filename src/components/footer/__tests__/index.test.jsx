import React from 'react';
import { render } from '@testing-library/react';
import Footer from '..';

describe('Footer', () => {
    it('should render the same snapshot when it is called', () => {
      const { asFragment } = render(<Footer />);
      expect(asFragment()).toMatchSnapshot();
    });
  });