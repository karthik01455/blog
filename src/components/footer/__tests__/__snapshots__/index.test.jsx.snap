// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render the same snapshot when it is called 1`] = `
<DocumentFragment>
  <div
    class="footer padding"
  >
    <p>
      Â©arifact.com 2019
    </p>
  </div>
</DocumentFragment>
`;
